openapiart/generator.py

```diff
@@ -1694,6 +1694,8 @@ def _get_openapi_types(self, yobject):
                 )
                 if min_max > 2147483647:
                     pt.update({key: r"'int64'"})
+                if min_max > 4294967294:
+                    pt.update({key: r"'uint64'"})
```

Comment:
I guess we're promoting types here ? Is this only applicable when format is provided ?
I believe it should only be applicable when format is provided.

I would suggest removing this change if this happens even when format has been provided.

Ideally - when format is not provided, we should deduce format based on min-max provided.

Reply:
done